src/store/store.ts

```diff
@@ -1,14 +1,13 @@
-import * as _ from "lodash";
+import _ from "lodash";
 import { autoinject } from "aurelia-framework";
 import { EventAggregator } from "aurelia-event-aggregator";
 import { Subscription } from "./messaging.model";
-
 const stateChanged = "state-changed:";
 
 @autoinject
-export class Store<TAppState> {
+export class Store<TAppState extends object> {
 
-	private state: TAppState;
+	private state!: TAppState;
```

Comment:
i think this make sense to be `TAppState | undefined` -- since until initialize it will be